scipy/sparse/tests/test_construct.py

```diff
@@ -802,16 +785,14 @@ def test_random_accept_str_dtype(self):
         construct.random_array((10, 10, 10, 10, 10), dtype='d')
 
     def test_random_array_maintains_array_shape(self):
-        # preserve use of old random_state during SPEC 7 transition
-        arr = construct.random_array((0, 4), density=0.3, dtype=int, random_state=0)
+        arr = construct.random_array((0, 4), density=0.3, dtype=int, rng=0)
         assert arr.shape == (0, 4)
 
         arr = construct.random_array((10, 10, 10), density=0.3, dtype=int, rng=0)
         assert arr.shape == (10, 10, 10)
 
-        arr = construct.random_array((10, 10, 10, 10, 10), density=0.3, dtype=int)
-        assert arr.shape == (10, 10, 10, 10, 10)
-
+        A = construct.random_array((10, 10, 10, 10, 10), density=0.3, dtype=int, rng=0)
+        assert A.shape == (10, 10, 10, 10, 10)
```

Comment:
I'd like to avoid naming changes so they don't distract in the main PR.
```suggestion
        arr = construct.random_array((10, 10, 10, 10, 10), density=0.3, dtype=int, rng=0)
        assert arr.shape == (10, 10, 10, 10, 10)
```